server/api/crud/model_monitoring/deployment.py

```diff
@@ -1038,15 +1038,22 @@ def _set_credentials_after_server_upgrade(self) -> bool:
         """
         credentials_dict = self._get_monitoring_mandatory_project_secrets()
         mm_enabled = False
-        store_connector: mlrun.model_monitoring.db.StoreBase = mlrun.model_monitoring.get_store_object(
-            project=self.project,
-            store_connection_string=credentials_dict.get(
+        store_connection_string = (
+            credentials_dict.get(
                 mlrun.common.schemas.model_monitoring.ProjectSecretKeys.ENDPOINT_STORE_CONNECTION
             )
-            or "v3io",  # in case the user use the default v3io
-        )
+            or "v3io"
```

Comment:
The "v3io" string occurs repeats in this module many times. I think it's worth a constant.